CHANGELOG.md

```diff
@@ -9,6 +9,39 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.5.0 - 2019-03-25
+
+### Added
+
+- Add "networkPolicyOnly" as a new "encapsulation mode".
```

Comment:
should we add more details to this ? like this mode allows Antrea to be responsible to enforce NetworkPolicies in EKS and GKE. something like that?

Reply:
I tried to clarify what that means, but this is not really the right place to document the differences between "noEncap" mode and "networkPolicyOnly" mode. And for GKE, we use "noEncap" mode.

Reply:
thats true.. GKE is not just networkPolicy but we replace kubenet.. so that makes sense.. so today this mode is only used in EKS.. 
thought about pointer to the doc explaining the policy only mode.. but it is still in review